pkg/accounts/model/repository.ts

```diff
@@ -9,6 +9,7 @@ export interface AccountRepository {
   create(account: Account): Promise<Result.Result<Error, void>>;
   findByName(name: string): Promise<Option.Option<Account>>;
   findByID(id: AccountID): Promise<Option.Option<Account>>;
+  findManyByID(id: AccountID[]): Promise<Result.Result<Error, Account[]>>;
```

Comment:
```suggestion
  findManyByID(id: readonly AccountID[]): Promise<Result.Result<Error, Account[]>>;
```
この配列の引数は変更されることを意図していないでしょうから, `readonly` を付けておきたいです.